CITATION.cff

```diff
@@ -0,0 +1,15 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: SpinalHDL
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
```

Comment:
@Readon @jijingg @kleinai Would you like to be added in the authors list ?

Reply:
I am glade to be listed :)

Reply:
Does it necessary to append the affliction?

Reply:
Is there any possible to add other contributors who submit issues and PRs, in short? like .etc?

Reply:
Ok for me ^^
But now idea if there is a field for it ?